README.md

```
@@ -24,16 +24,19 @@ A TypeScript-first client for seamless integration with the [SMSGate](https://sm
   - [🚀 Quickstart](#-quickstart)
     - [Basic Usage](#basic-usage)
     - [Webhook Management](#webhook-management)
+    - [Webhook Events](#webhook-events)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)